packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -7153,7 +7153,83 @@ class KeyedSubtree extends StatelessWidget {
   Widget build(BuildContext context) => child;
 }
 
-/// A platonic widget that calls a closure to obtain its child widget.
+/// A platonic (simple) widget that provides a specific [BuildContext] object to its child
+/// widget.
+///
+/// This [BuildContext] object is unique and is not the same object as
+/// the one provided by the framework in the `Widget build` method.
```

Comment:
"Widget build" method should not be in `. Also, Widget does not have a build method.